src/integration/pages/frontPage/__snapshots__/arabic.test.js.snap

```diff
@@ -3068,7 +3068,6 @@ exports[`Given I am on a Arabic Frontpage AMP/Canonical page When I view the sou
                       </a>
                     </h3>
                     <p class="Summary-sc-1dvfmi3-4 gZhHSq">
-                      اشتباكات بين متظاهرين والشرطة الإسرائيلية، وامرأة تسقط أثناء مشيها على الحبل، من بين أفضل صور الأسبوع.
```

Comment:
Hmm, why has all of the summary text disappeared in the integration snapshots?

Reply:
The `overtypedSummary`'s in our data directory are not empty strings since they contain white space. https://github.com/bbc/simorgh/blob/f6710d8b797dd92224c9c31e210f9df73292c5be/data/arabic/frontpage/index.json#L2145
https://github.com/bbc/simorgh/blob/f6710d8b797dd92224c9c31e210f9df73292c5be/data/arabic/frontpage/index.json#L2265



This will result in `overtypedSummary` being `true` in https://github.com/bbc/simorgh/blob/e8fd9c9253cac70579e9d334386596feff39e08b/src/app/containers/StoryPromo/index.jsx#L110

Reply:
Thanks for flagging this @EinsteinNjoroge, will add another check for the white space.